INSTALL.md

```diff
@@ -0,0 +1,268 @@
+# Installing google-cloud-cpp
+
+By default google-cloud-cpp libraries download and compile all their
+dependencies. This makes it easier for users to "take the library for a spin",
+and works well for users that "Live at Head", but does not work for package
+maintainers or users that prefer to compile their dependencies once and install
+them in `/usr/local/` or a similar directory.
+
+This document provides instructions to install the dependencies of
+google-cloud-cpp.
+
+## Installing google-cloud-cpp
+
+*If*<br>
+-- The Spartan ephors to Philip of Macedon.<br><br>
```

Comment:
What purpose does this serve?

Reply:
Remind myself and the readers that some "ifs" are more equal than others.  Removed.

Reply:
Gotcha, more of a disclaimer that it might not work in all cases. I thought it's a little distracting, thanks for removing it.